bookkeeper-benchmark/src/main/java/org/apache/bookkeeper/benchmark/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+/**
+ * @TODO: Write JavaDoc comment
```

Comment:
@sijie, At the moment there are 25 TODOs added in this PR. There are also 25 TODOs added in the #234 PR. But the bookkeeper-server isn't finished.
I think it is not a huge number of TODOs, but it takes a while to write the proper documentation.
It was conspicuous only because those are in the same PR, but (25+25) 50 TODOs are not so many in a huge project like this.